server/server.js

```diff
@@ -52,4 +52,15 @@ app.delete('/api/hostels/:hostel_id/images', (req, res) => {
   });
 });
 
+app.put('/api/hostels/:hostel_id/images', (req, res) => {
+  console.log(req.body);
+  models.image.updateEntry(req.body, (error, updatedImage) => {
+    if (error) {
+      res.status(502).send();
+      return;
+    }
+    res.send(updatedImage);
```

Comment:
You can probably just send a message, the front end won't be using the data from a put call to render.